.nyc_output/processinfo/index.json

```diff
@@ -0,0 +1,75 @@
+      "children": ["103bccc2-353c-4f4a-aaae-362a2c4c249b"]
+    }
+  },
+  "files": {
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/index.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/routes/v1/index.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/routes/v1/auth.route.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/helpers/responseGenerator.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/helpers/utils.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/repository/base.repository.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/database/models/index.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/database/config/config.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/database/models/follower.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/database/models/user.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/routes/v1/user.route.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/middleware/users.middleware.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/controllers/user.controller.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/helpers/pagination.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/helpers/utils/mailer.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/helpers/generateToken.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/middleware/validation.middleware.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/middleware/validation.schema.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/middleware/auth.middleware.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ],
+    "/Users/olorunwalawrence/Desktop/ah-kifaru-backend/src/middleware/pagination.validation.js": [
+      "103bccc2-353c-4f4a-aaae-362a2c4c249b"
+    ]
+  },
+  "externalIds": {}
+}
```

Comment:
Well-done! You may want to remove this file as said previously. @olorunwalawrence